vyper/ast/pre_parser.py

```diff
@@ -155,7 +155,11 @@ def consume(self, token, result):
 # simple statements that are replaced with `yield`
 CUSTOM_STATEMENT_TYPES = {"log": "Log"}
 # expression types that are replaced with `await`
-CUSTOM_EXPRESSION_TYPES = {"extcall": "ExtCall", "staticcall": "StaticCall"}
+CUSTOM_EXPRESSION_TYPES = {
+    "extcall": "ExtCall",
+    "staticcall": "StaticCall",
+    "staticall": "BadStaticCall",  # common typo - we parse it to AST for better error reporting
```

Comment:
some other (not so common, but still) typos are:
- `staticcal` (that one we should probably also cover)
- `statiiccall`
- `stacticall`
- `staticcal`

Reply:
although I can imagine such mistakes being made, I don't like the idea that we would reserve 5 keywords just for error reporting (though I don't think such keywords would be used by users anyway)